doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1050,6 +1050,7 @@ I/O
 - Bug in :meth:`~HDFStore.create_table` now raises an error when `column` argument was not specified in `data_columns` on input (:issue:`28156`)
 - :meth:`read_json` now could read line-delimited json file from a file url while `lines` and `chunksize` are set.
 - Bug in :meth:`DataFrame.to_sql` when reading DataFrames with ``-np.inf`` entries with MySQL now has a more explicit ``ValueError`` (:issue:`34431`)
+- Bug in :meth:`io.common.infer_compression` where capitalised files extensions were not decompressed by read_* functions.
```

Comment:
can you add this PR number here (use the issue format). also add 'on windows platforms'

Reply:
@jreback changed in a3558fd